conf/application.conf

```diff
@@ -45,36 +45,46 @@ play.http.errorHandler = "handlers.ErrorHandler"
 
 microservice {
   services {
-      auth {
-        protocol = http
-        host     = localhost
-        port     = 8500
-      }
-      emcs-tfe-cancel-movement-frontend {
-        protocol = http
-        host = localhost
-        port = 8318
-      }
-      emcs-tfe {
-        protocol = http
-        host = localhost
-        port = 8311
-      }
-      user-allow-list {
-        protocol = http
-        host = localhost
-        port = 11306
-      }
-      reference-data {
-        protocol = http
-        host = localhost
-        port = 8312
-      }
-        contact-frontend {
-          protocol = http
-          host = localhost
-          port = 9250
-        }
+    auth {
+      protocol = http
+      host     = localhost
+      port     = 8500
+    }
+    emcs-tfe-cancel-movement-frontend {
+      protocol = http
+      host = localhost
+      port = 8318
+    }
+    emcs-tfe {
+      protocol = http
+      host = localhost
+      port = 8311
+    }
+    user-allow-list {
+      protocol = http
+      host = localhost
+      port = 11306
+    }
+    reference-data {
+      protocol = http
+      host = localhost
+      port = 8312
+    }
+    contact-frontend {
+      protocol = http
+      host = localhost
+      port = 9250
+    }
+    emcs-tfe-reference-data {
+      protocol = http
+      host = localhost
+      port = 8312
+    }
```

Comment:
emcs-tfe-reference-data is the same as reference-data further up, so either `emcs-tfe-reference-data` or `reference-data`  can be removed